modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,16 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} with vMotion
+
+{product-title} generally supports compute-only vMotion but not Storage vMotion.
+
+If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. This prevents affected pods from starting up and can result in data loss. Therefore, any form of Storage vMotion is unsupported.
+
```

Comment:
@bobfuru leaving this to you but I would perhaps make "storage vMotion is unsupported" text bold or something. so people don't ignore this right off the bat.

Reply:
I wrapped the first paragraph in a note style to call this out right away.